config/invenio.conf

```diff
@@ -1942,6 +1942,12 @@ CFG_BATCHUPLOADER_WEB_ROBOT_RIGHTS = {
 # By default all DOIs are excluded
 CFG_BIBUPLOAD_INTERNAL_DOI_PATTERN = [^\w\W]
 
+# CFG_BIBUPLOAD_DOI_CHECKING - check multi DOIs
+# Parameter for switch on / off DOI checking in bibuploads
+# CFG_BIBUPLOAD_DOI_CHECKING = 0: multi DOIs are possible
+# CFG_BIBUPLOAD_DOI_CHECKING = 1: multi DOIs are forbidden (default)
+CFG_BIBUPLOAD_DOI_CHECKING = 1
```

Comment:
Moreover, the explanation for values can come simply in the comment, so that the variable name is not necessarily repeated. For example:

```
# CFG_BIBPLOAD_CHECK_DOI -- do we want to check for 
# DOI uniqueness during upload jobs? (True/False)
CFG_BIBUPLOAD_CHECK_DOI = True
```